mule-gateway/modules/ROOT/pages/mule-gateway-capabilities-mule4.adoc

```diff
@@ -137,6 +137,58 @@ Depending on your implementation, there are several ways to pass organization cr
 See xref:api-manager::org-credentials-config-mule4.adoc[Configuring Organization Credentials in Mule Runtime 4] or xref:api-manager::org-credentials-config-mule3.adoc[Mule Runtime 3] to learn how to do this.
 
 
+[[mule-gateway-hyperforce]]
+== Mule Gateway on Hyperforce
+
+All features of Mule Gateway are supported on Hyperforce with these exceptions:
+
+* API Alerts
+* Key Metrics
+
+For more information about API Manager's support on Hyperforce, see xref:api-manager::latest-overview-concept.adoc#APIM-hyperforce[API Manager on Hyperforce].
+
+=== Mule Gateway Proxies on Hyperforce
+
+All Mule Gateway proxies are supported on Hyperforce. See the following table to find the first proxy versions that are supported on Hyperforce:
+
+[%header%autowidth.spread]
+|===
+|Policy |Version
+| HTTP Proxy |2.1.1
+| HTTPS Proxy |2.1.1
+| REST Proxy |3.2.2 and 2.2.1
+| WSDL Proxy |3.1.3 and 2.2.1
+|===
```

Comment:
First versions actually are:
- HTTP and HTTPS Proxy: 2.1.0
- REST: 2.2.1, 3.2.1
- WSDL: 2.2.1, 3.1.1